### DISCUSSION TITLE
Zotero Import Error: Templater skipped parsing XXXXX.md because file size exceeds 10000

### DISCUSSION THREAD

**Discussion by Asstrole** [SilentVoid13/Templater#1314] [Ideas & Feature Requests]
Hey there. I am running into the following error in the console: **Templater skipped parsing XXXXXXX.md because file size exceeds 10000**

This occurs while using Zotero Integration Plugin (Obsidian) along with Templater. After executing an Import command using the Zotero Plugin, a markup file is loaded as an import template. Zotero Plugin supports Nunjucks for the markup file. Within the import markup file is the below Templater command. The error is only thrown for an import (PDF annotations) that has a great deal of annotations to parse. Typically no issues except for large PDFs. When this error is thrown the console and rename portion of the below command does not fire. Is there a command/setting to increase this Templater limitation?

```
<%*
let title = "{{title}}";
title = title.replace(/[:\/\\|*<>?"]/g, "•");
console.log(title);
let date = tp.date.now("YYYY-MM-DD");
await tp.file.rename(`& ${date} ${title}`)
;_%>
```
Thank you for any guidance!

**Comment by Zachatoo** [comment]
This was added due to an issue reported with large files causing Templater to freeze up Obsidian, see #873 and #315

There's not currently a way to change that limit, though it's certainly a feature that could be added

**Comment by Asstrole** [comment]
Thank you much Zachatoo
